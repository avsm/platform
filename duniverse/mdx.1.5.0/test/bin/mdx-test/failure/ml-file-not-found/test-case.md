Run this with `--direction=to-md`

```ocaml file=not_found.ml
```
