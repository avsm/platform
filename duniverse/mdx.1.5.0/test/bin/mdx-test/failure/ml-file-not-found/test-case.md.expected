Error in the OCaml code block in test-case.md at line 4:
./not_found.ml: No such file or directory
